app/templates/gentelella/users/events/_table.html

```diff
@@ -1,6 +1,23 @@
 {% macro events_table(events) %}
-    <div class="table-responsive">
-        <table class="table table-striped">
+
+<script src="{{ url_for('static', filename='vendor/datatables.net/js/jquery.dataTables.min.js') }}"></script>
+<script src="{{ url_for('static', filename='vendor/datatables.net-bs/js/dataTables.bootstrap.min.js') }}"></script>
+<script type = "text/javascript" >
+	 $('#dodo').DataTable({
```

Comment:
@niranjan94 have reduced repetition to 2 by commenting 
<script src="{{ url_for('static', filename='vendor/datatables.net/js/jquery.dataTables.min.js') }}"></script> <script src="{{ url_for('static', filename='vendor/datatables.net-bs/js/dataTables.bootstrap.min.js') }}"></script>  
in the app/templates/gentelella/users/events/index.html  as there was as such no use of datables in that.